src/allmydata/scripts/runner.py

```diff
@@ -194,7 +194,49 @@ def run():
     # doesn't return: calls sys.exit(rc)
     task.react(_run_with_reactor)
 
+
+def _setup_coverage(reactor):
+    """
+    Arrange for coverage to be collected if the 'coverage' package is
+    installed
+    """
+    # can we put this _setup_coverage call after we hit
+    # argument-parsing?
+    if not '--coverage' in sys.argv:
+        return
+    sys.argv.remove('--coverage')
+
+    try:
+        import coverage
+    except ImportError:
+        return
```

Comment:
If coverage was requested should we silently not collect it?  Does letting this "error" pass here make some other part of the system (CI configuration, tox, whatever) easier?  Or would it be okay to blow up right here saying we can't collect coverage?